tests/ImageSharp.Tests/Formats/Jpg/JpegEncoderTests.cs

```diff
@@ -55,20 +56,27 @@ public void EncodeBaseline_IsNotBoundToSinglePixelType<TPixel>(TestImageProvider
             TestJpegEncoderCore(provider, subsample, quality);
         }
 
-        private static ImageComparer GetComparer(int quality)
+        /// <summary>
+        /// Anton's SUPER-SCIENTIFIC tolerance threshold calculation
+        /// </summary>
+        private static ImageComparer GetComparer(int quality, JpegSubsample subsample)
```

Comment:
The scaled approach is the same I found in the libjpeg source so I wonder why it would produce such different output? Happy for you to replace it in the future with your SIMD wizardry though!

Reply:
Its not a proper correctness testing for now, I'm comparing the orinal image against the one that's saved by our encoder then decoded by S.D. The goal was to cath regressions.

However  there are a few results with 444+Q=100 encoding which should be better IMO. Libjpeg can is able to choose from several code paths AFAIK depending on configuration + architecture, LUT is just one of them and SIMD is preferred whenever available. (At least the decoder works this way.)